### PR TITLE
Deploy app

### PR DESCRIPTION
**Pull request by STEPHINRACHEL**
**What**
Deploying the app on an existing cluster

**To Verify**
Follow the steps in README.md under Deploying the Application on the cluster

### CODE REVIEW

**Comment by STEPHINRACHEL** [comment]
@laurafitzgerald Had some issue in pushing to the repo from the old branch and created the new PR from the fork.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,19 @@
 # workload-web-app
 A test app for simulating the workload on the openshift cluster based on end-user use cases in order to monitor the downtime of component products in integreatly during an upgrade.
+
+
+## Deploying the Application on the cluster
+
+To deploy the webapp on your cluster:
+
+> Note: 
+🍎 (Mac users) - install gtimeout util and create a symbolic link (so it can be referenced as timeout):
```
Comment (laurafitzgerald) [inline README.md:10]: 👍

**Comment by laurafitzgerald** [comment]
It works for me. Happy to have others review as well.
/lgtm

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+test:
+	@echo "SUCCESS"
+
+.PHONY: deploy
+deploy:
+	-@oc new-project $(NAMESPACE)
+	@oc apply -f deploy/dc/deployment_config.yaml
+	@oc apply -f deploy/svc/service.yaml
+	@oc expose -f deploy/svc/service.yaml
+	$(call wait_command, oc get dc -n $(NAMESPACE) -o json | jq '.items[] | .status |.readyReplicas' | grep 1 , available replicas, 5m, 30)
```
Comment (wei-lee) [inline Makefile:19]: you can use `oc wait` which can do the same thing. save the trouble of installing `timeout`

**Comment by wei-lee** [comment]
/lgtm
added a small suggestion to use `oc wait`

**Comment by STEPHINRACHEL** [comment]
/approve 
Merging in order to unblock other related tasks

**Pull request merged**